bin/doctor

```diff
@@ -185,11 +185,11 @@ function check_config_files() {
         source "$TOOLKIT_ROOT/$config_file"
 
         # Check some vars from the RC file
-        if [[ "${SHARELATEX_DATA_PATH:-null}" != "null" ]]; then
-          print_point 2 "SHARELATEX_DATA_PATH: $SHARELATEX_DATA_PATH"
+        if [[ "${OVERLEAF_DATA_PATH:-null}" != "null" ]]; then
```

Comment:
4.x setups are still allowed to use the SHARELATEX_ prefixed ones. We could add a fallback like I did in https://github.com/overleaf/toolkit/pull/221

Reply:
After upgrading the toolkit they should have these migrated to `OVERLEAF_` prefix. We can print both, and also print information in the doctor about whether their successfully migrated their `overlearf.rc`.